models/user.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+const { Model } = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    static associate(models) {}
+  }
+  User.init(
+    {
+      id: {
+        field: "user_uuid",
+        type: DataTypes.UUID,
+        primaryKey: true,
```

Comment:
You also need `autoIncrement: true` or Sequelize will pass NULL and that will bypass letting the db generate the uuid.

Reply:
Will do, I think the primary key assignment is a bit murky for me.

Once the sequelize generated query hits the DB, pgcrypto generates an `id` for the primary key right? Is the auto-increment flag intended to have sequelize create a "placeholder" id so our query passes the NOT NULL validation?

Reply:
The autoIncrement flag tells sequelize to not pass ANYTHING for the id field- that allows the db to generate the id for the column based on the default function.

Reply:
Ok interesting. Not what I would have expected but I got it.